jarviscli/plugins/news.py

```diff
@@ -58,22 +60,71 @@ def run(self, jarvis, s):
                 jarvis.say("I couldn't find news", Fore.RED)
         else:
             try:
+                if len(jarvis.get_data('news-settings')) > 0:
```

Comment:
Crashes if news are not configured:
```
Traceback (most recent call last):
  File "/home/phil/git/Jarvis/jarviscli/CmdInterpreter.py", line 143, in try_do
    do(s)
  File "/home/phil/git/Jarvis/jarviscli/plugin.py", line 90, in _plugin_run_with_network_error
    run_func(jarvis, s)
  File "/home/phil/git/Jarvis/jarviscli/plugins/news.py", line 63, in run
    if len(jarvis.get_data('news-settings')) > 0:
TypeError: object of type 'NoneType' has no len()
```

Probably missing a ``is not None`` check.

Reply:
None check has been implemented